docs/reference/modules/scripting.asciidoc

```diff
@@ -24,23 +24,18 @@ and give the most flexibility.
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Language
-    |Sandboxed
     |Required plugin
```

Comment:
Perhaps we use the term Sandboxed but add a note explaining what we mean, eg:

    _Sandboxed_ languages are those that can be enabled safely, without security risks.  
    Languages like Lucene Expressions, Painless, and Mustache have controls in place
    to ensure that a user can only access safe operations.  A language like Groovy allows 
    the user too much flexibility, which can pose a security risk.  Even a simple
    `while (true) {...}` loop in Groovy can be used for a denial of service attack.

... or something better :)